Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: lint
+lint:
+	pylint dast --max-line-length=120
+	mypy . --ignore-missing-imports
```

Comment:
why we ignore missing imports?

Reply:
Not all packages include types unfortunately and they are errors with mypy. If you try to run `make lint` without it you'll see the errors are not very helpful and nothing we can do anything about

Reply:
:pain: